src/tools/perf/libperf.c

```diff
@@ -1503,7 +1520,8 @@ ucs_status_t ucx_perf_run(ucx_perf_params_t *params, ucx_perf_result_t *result)
     ucx_perf_test_reset(perf, params);
 
 #if HAVE_CUDA
-    if (params->mem_type == UCT_MD_MEM_TYPE_CUDA) {
+    if (params->mem_type == UCT_MD_MEM_TYPE_CUDA
```

Comment:
Please guard each statement with () and move || to the first line

Reply:
@yosefe this doesn't apply. There's just one condition